Kernel/Devices/SerialDevice.cpp

```diff
@@ -86,12 +87,12 @@ void SerialDevice::set_baud(Baud baud)
     m_baud = baud;
 
     IO::out8(m_base_addr + 3, IO::in8(m_base_addr + 3) | 0x80); // turn on DLAB
```

Comment:
If we want to be more correct about this (so we use `IOAddress`), let's use `m_base_addr.out<u8>()` styled calls instead.

Reply:
Good idea, done!